Spinner/Spinner.js

```diff
@@ -55,7 +61,7 @@ const SpinnerCore = kind({
 	},
 
 	defaultProps: {
-		type: 'searching'
+		type: SPINNER_TYPE.searching
```

Comment:
I'm not following the logic behind converting this to a constant object. The type should just be specified as `PropTypes.oneOf('searching', 'loading'), and we can refer to these strings like normal throughout the component. If we're concerned about translations, put the `$L` method in the aria-label area and let everything be simpler.